CHANGELOG.md

```diff
@@ -2,9 +2,10 @@
 
 ### Features
 
-- `[feat(@jest/environment, jest-runtime)]` Allow `jest.requireActual` and `jest.requireMock` to take a type argument ([#13253](https://github.com/facebook/jest/pull/13253))
```

Comment:
Wanna send a separate PR for these to changes? That way CI will trigger when you push without me having to press a button 🙂

Reply:
I'm not totally sure what you mean, maybe suggest a change?

Reply:
Extract the removal of `feat` from the changelog to a separate PR

Reply:
I tried doing that and the workflow did not automatically run
https://github.com/facebook/jest/pull/13268/commits/8969360d91fd992810cf19874163b507133dea2b

Reply:
Yeah, I meant make those changes in a separate PR which I'll merge. Then approval is no longer needed (once you have a merged contribution)

Reply:
sure